aerogear/mobile-security-service
# feat(ui): Created a modal to make users confirm they wish to save an app (AEROGEAR-8472)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-8472

## What

Create a confirmation modal that will be shown to the user to confirm they wish to save an app.

## Why

It is good practice to make a user confirm an action such as modifying critical server data.

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Navigate to the app detailed page.
2. Click `Save App`.
3. A confirmation modal will appear.
## Checklist:


## Progress
